tools/bin/check_images_exist.sh

```diff
@@ -1,78 +1,133 @@
 #!/usr/bin/env bash
 
-set -e
+# ------------- Import some defaults for the shell
+
+# Source shell defaults
+# $0 is the currently running program (this file)
+this_file_directory=$(dirname $0)
+relative_path_to_defaults=$this_file_directory/../shell_defaults
+
+# if a file exists there, source it. otherwise complain
+if test -f $relative_path_to_defaults; then
+  # source and '.' are the same program
+  source $relative_path_to_defaults
+else
+  echo -e "\033[31m\nFAILED TO SOURCE TEST RUNNING OPTIONS.\033[39m"
+  echo -e "\033[31mTried $relative_path_to_defaults\033[39m"
+  exit 1
+fi
+
+set +o xtrace  # +x easier human reading here
 
 . tools/lib/lib.sh
 
+
 function docker_tag_exists() {
   # Added check for images pushed to github container registry
   if [[ $1 == ghcr* ]]
```

Comment:
Is there somewhere that explains what the args for this script are supposed to be?

Reply:
Nope, I didn't write this incantation, but it's a good note since I understand it now.  

I'll update the PR, but there are two places images live dockerhub and GitHub... something that starts with those letters